core/server/worker/src/main/java/alluxio/worker/grpc/GrpcExecutors.java

```diff
@@ -70,12 +71,31 @@
           new ImpersonateThreadPoolExecutor(BLOCK_WRITE_THREAD_POOL_EXECUTOR);
 
   static {
+    MetricsSystem.registerGaugeIfAbsent(MetricsSystem.getMetricName(
+            MetricKey.WORKER_CACHE_MANAGER_THREAD_ACTIVE_COUNT.getName()),
+        CACHE_MANAGER_THREAD_POOL_EXECUTOR::getActiveCount);
+    MetricsSystem.registerGaugeIfAbsent(MetricsSystem.getMetricName(
+            MetricKey.WORKER_CACHE_MANAGER_THREAD_CURRENT_COUNT.getName()),
+        CACHE_MANAGER_THREAD_POOL_EXECUTOR::getPoolSize);
+    MetricsSystem.registerGaugeIfAbsent(MetricsSystem.getMetricName(
+            MetricKey.WORKER_CACHE_MANAGER_THREAD_QUEUE_WAITING_TASK_COUNT.getName()),
+        CACHE_MANAGER_THREAD_POOL_EXECUTOR.getQueue()::size);
+    MetricsSystem.registerGaugeIfAbsent(MetricsSystem.getMetricName(
```

Comment:
I think this is a static value that does not change. So there's no need to monitor it.

Reply:
see above